.github/workflows/pull-e2e-tests.yml

```
@@ -0,0 +1,26 @@
+name: e2e-tests
+
+on: [pull_request]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  test-e2e:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '>=1.20'
+          check-latest: true
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+      - run: make deps-frontend frontend deps-backend
+      - run: npx playwright install --with-deps
+      - name: run tests
+        run: timeout -s ABRT 40m make test-e2e-sqlite
```

```suggestion
        run: make test-e2e-sqlite
        timeout-minutes: 40
```
We could use the native timeout functionality from GitHub: [timeout-minutes](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes)

---

Will do, it's in use in 6 places total.

---

done, remove names there as well now